### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by ttytm**
## Description

Please write a few sentences describing the overall goals of the pull request's commits.

## PR checklist

Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")

- [x] **changelog** not required
- [x] **3.6 backport** done, or not required
- [x] **2.28 backport** not required (Testing code is not present in LTS)
- [x] **tests** not required

## Notes for the submitter

Please refer to the [contributing guidelines](https://github.com/Mbed-TLS/mbedtls/blob/development/CONTRIBUTING.md), especially the
checklist for PR contributors.

Help make review efficient:
* Multiple simple commits
  - please structure your PR into a series of small commits, each of which does one thing
* Avoid force-push
  - please do not force-push to update your PR - just add new commit(s)
* See our [Guidelines for Contributors](https://mbed-tls.readthedocs.io/en/latest/reviews/review-for-contributors/) for more details about the review process.

### CODE REVIEW

File: scripts/generate_ssl_debug_helpers.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def __str__(self):
             {translation_table}
                 }};
 
-                return "UNKOWN";
+                return "UNKNOWN";
```
Comment (tom-cosgrove-arm) [inline scripts/generate_ssl_debug_helpers.py:331]: This was previously fixed in #5732 at https://github.com/Mbed-TLS/mbedtls/pull/5732/files#diff-c1d4f10f0fef89231d91756c0b67ee38bcbed022a4b80f0ea3b44fb4c0e4b845R292, so how come it's wrong in-tree?

Comment (ttytm) [reply scripts/generate_ssl_debug_helpers.py:331]: 47854e638b48de67d7daa195b9ebc2f0cbd1eec6 reverted `return "UNKOWN_VAULE";` to the  return with the oopsi.

Comment (tom-cosgrove-arm) [reply scripts/generate_ssl_debug_helpers.py:331]: Ah yes, but for "UNKOWN" we only fixed one of two instances in the file back in #5732 - there is so much duplicated template around this line that it wasn't obvious to me yesterday evening

**Review by gilles-peskine-arm — APPROVED** [review]
LGTM

**Pull request merged**